File: CSharp/SistemaAcademico/Models/Estudiante.cs
using System.ComponentModel.DataAnnotations;

namespace SistemaAcademico.Models;
public class Estudiante {
    public int Id { get; set; }
    public required string Nombre { get; set; }
    [MaxLength(15)]
    public required string Cedula { get; set; }

    // Relación Muchos a Muchos
    public List<Materia> Materias { get; set; } = [];
}

File: PHP/mi_proyecto/app/Models/Estudiante.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsToMany;

class Estudiante extends Model
{
    protected $fillable = ['nombre', 'cedula'];

    // Relación: Un estudiante se inscribe en muchas materias (Equivalente al ManyToManyField)
    public function materias(): BelongsToMany
    {
        return $this->belongsToMany(Materia::class, 'estudiante_materia');
    }
}